crossbeam-epoch/src/internal.rs

```diff
@@ -35,10 +35,11 @@
 //! Ideally each instance of concurrent data structure may have its own queue that gets fully
 //! destroyed as soon as the data structure gets dropped.
 
-use core::cell::{Cell, UnsafeCell};
+use crate::concurrency::cell::UnsafeCell;
```

Comment:
UnsafeCell is not technically a concurrent thing.  Is it logically correct to have it under `crate::concurrency`?

Reply:
I agree, but `concurrency` is the module that holds all the stuff that differs between the loom and non-loom contexts, and it felt weird to break out a new, dedicated top-level module just for `UnsafeCell`. We could rename `concurrency` to something else, but I can't immediately think of something better? And to give a (weak) defense for the current scheme, `UnsafeCell` does _usually_ come up in the context of concurrency where you want to unsafely assert that you can access a given type that is technically shared.

Reply:
Hmm, yeah, it's really anything that ends up being a synchronization point. We could use `synchronization`?